package/suc/run.sh

```diff
@@ -5,6 +5,26 @@ set -x -e
 CATTLE_AGENT_VAR_DIR=${CATTLE_AGENT_VAR_DIR:-/var/lib/rancher/agent}
 TMPDIRBASE=${CATTLE_AGENT_VAR_DIR}/tmp
 
+if [ -f /host${CATTLE_AGENT_VAR_DIR}/env/CATTLE_CACERTS_VALID ]; then
+  export CATTLE_CACERTS_VALID="$(cat /host${CATTLE_AGENT_VAR_DIR}/env/CATTLE_CACERTS_VALID)"
```

Comment:
The path has to match with https://github.com/rancher/rancher/pull/48569/files#diff-2d6a56c7215bb70f2f4af658fd18d81ccf71dd7774f92c37d987a608703515cfR286, so let me know if this needs to be changed. 

Also I can remove the logging, but I did use it to verify that the code was working.